zerver/lib/integrations.py

```diff
@@ -135,13 +140,13 @@ class GithubIntegration(WebhookIntegration):
     We want to have one generic url with dispatch function for github service and github webhook.
     """
     def __init__(self, name, client_name=None, logo=None, secondary_line_text=None,
-                 function=None, url=None, display_name=None, doc=None):
-        # type: (str, Optional[str], Optional[str], Optional[str], Optional[str], Optional[str], Optional[str], Optional[str]) -> None
+                 function=None, url=None, display_name=None, doc=None, stream_name=None):
+        # type: (str, Optional[str], Optional[str], Optional[str], Optional[str], Optional[str], Optional[str], Optional[str], Optional[str]) -> None
         url = self.DEFAULT_URL.format(name='github')
 
         super(GithubIntegration, self).__init__(
             name, client_name, logo, secondary_line_text,
-            function, url, display_name, doc)
+            function, url, display_name, doc, stream_name)
```

Comment:
This is a TODO for a future cleanup, but I'm not really a fan of passing a bunch of kwargs through this way, since there's a lot of risk of screwing up adding a new one and ending up with arguments swapped.